Lighthouse Camera/Storyboards/LaunchScreen.storyboard

```diff
@@ -16,12 +20,21 @@
                     <view key="view" contentMode="scaleToFill" id="Ze5-6b-2t3">
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
+                        <subviews>
```

Comment:
Correct me if I'm wrong, but I had the impression that `LaunchScreen` is displayed while the app is loading, right? If so, I don't quite understand this part of the patch.